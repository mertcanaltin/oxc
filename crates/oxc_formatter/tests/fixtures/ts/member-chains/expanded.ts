const defaultColorDecoratorsEnablement = accessor
  .get(IConfigurationService)
  .getValue<'auto' | 'always' | 'never'>('longlonglonglonglonglonglonglonglong');
