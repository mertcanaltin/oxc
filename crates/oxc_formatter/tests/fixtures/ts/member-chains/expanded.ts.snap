---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
const defaultColorDecoratorsEnablement = accessor
  .get(IConfigurationService)
  .getValue<'auto' | 'always' | 'never'>('longlonglonglonglonglonglonglonglong');

==================== Output ====================
const defaultColorDecoratorsEnablement = accessor
  .get(IConfigurationService)
  .getValue<
    "auto" | "always" | "never"
  >("longlonglonglonglonglonglonglonglong");

===================== End =====================
